src/helper.js

```diff
@@ -0,0 +1,29 @@
+const getErrors = (givenText, givenCase) => {
+  const acceptableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Create this variable outside the function to avoid re-creating it every time when the function is executed